starter-ui/src/main/java/org/eclipse/starter/ui/Main.java

```diff
@@ -0,0 +1,93 @@
+package org.eclipse.starter.ui;
```

Comment:
This looks pretty involved. I think let's first see if we can just install Tomcat on the server. That way, all we need to do is build a very simple war. It will eliminate all this code.

Reply:
True, I wanted to run from the main method. I will simplify it. In that case, I'll have to write a good readme how to run it.